tests/test_catalog.py

```diff
@@ -1,19 +1,18 @@
 import json
 import os
+import pathlib
```

Comment:
Just for curiosity, Why did we make this change?

Reply:
https://github.com/up42/up42-py/blob/e0e0e13e56fde573e9ce9ab6a056af222827710e/pyproject.toml#L52